doc/pmon/pmon-chassis-design.md

```diff
@@ -358,6 +356,29 @@ Changes in thermalctld would follow one of the 2 approaches:
 1. Have a TemperatureUpdater class for each line-card. Each of the updater class will fetch the values for all temperature senosors of the line-card from the REDIS-DB of the line-card and update the DB on the control-card.
 2. The TemperatureUpdater class in each line-card will update the local-DB on its card as well as the global-DB on the control-card.
 
+We have chosen to take Approach 2. A GLOBAL_STATE_DB will be added in the chassis-redis instance similar to the STATE_DB in the local-redis instance.
+
+#### GLOBAL_STATE_DB Schema
+```
+key                                   = TEMPERATURE_INFO_<card-index> | <Sensor-Name>; 
+; field                               = value
+critical_high_threshold               = float;
+critical_low_threshold                = float;   
+high_threshold                        = float;
+low_threshold                         = float;
+maximum_temperature                   = float; //Maximum recorded temperature
+minimum_temperature                   = float; //Minimum recorded temperature
+temperature                           = float;
+
+```
```

Comment:
Where is temperature sensors polling interval defined?
Once Voltage and current sensors are taken into account, they would have their respective polling intervals.
Usually polling interval of temperature sensors is aggressive than voltage/current sensors polling interval.

Request to sync up/ discuss  and take these into consideration

Reply:
This would be an Infra item to take up across processes. Have added in Future Items section.